.directory

```diff
@@ -0,0 +1,4 @@
+[Desktop Entry]
```

Comment:
Here and elsewhere, this PR includes files which do not belong in it, please have a look at the resulting list of diffs before marking a PR ready for review.